src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -73,81 +60,95 @@ You can view a deeper layer of granularity in your metrics by clicking on any me
-
-<img
-    style={{ align: 'left',maxWidth: '90%' }}
-    title="Screenshot showing attributes behind metrics"
-    alt="Screenshot showing attributes behind metrics"
-    src="/images/apis-and-data_screenshot-crop_metric-attributes.webp"
-/>
-
-This information is only for the last day (since UTC midnight), which is when cardinality resets (both account and metric). 
-
-You can also view specific information about an attribute, which can help you make decisions about whether or not that attribute is useful. You can see a sample of the attribute values by clicking the menu on the right side of the attribute row.
-
+The <DNT>**Attributes**</DNT> table has the list of top 20 attributes that majorly contribute to the metric's cardinality. To view the values of a specific attribute, click the options menu on the right side of the attribute row and select <DNT>**View attribute values**</DNT>.
 
 ## Account pruning rules [#account-pruning-rules]
 
-A common way to keep account cardinality under its limit is to implement pruning rules for metrics and attributes you don't consider important. 
+A common method to maintain account cardinality within its limit is to implement pruning rules for non-critical attributes, which removes unnecessary data.
 
 ### View account pruning rules [#view-pruning-rules]
 
-You can view pruning rules by clicking into a metric and then clicking **View Pruning Rules**. This will bring up a scrollable list of all pruning rules associated with an account. You can also delete pruning rules associated with an account in this view:
-
+To view account pruning rules:
+1. Go to the **Cardinality Management** page of the account.
+2. From the **Metrics** table, select <DNT>**View Pruning Rules**</DNT>. A scrollable list of all pruning rules associated with the account appears.
+  <Callout variant="tip">
+If no pruning rules exist in the account, the <DNT>**View Pruning Rules**</DNT> button doesn't appear.
+</Callout>
 <img
     style={{ align: 'left',maxWidth: '90%' }}
     title="Screenshot showing the link to pruning rules on the Attributes page"
     alt="Screenshot showing the link to pruning rules on the Attributes page"
     src="/images/apis-and-data_screenshot-crop_view-pruning-rules.webp"
 />
 
-
 ### Create a pruning rule [#create-pruning-rule]
 
-You can create a pruning rule based on attributes within the UI. To create a pruning rule:
+To create a pruning rule:
 
-1. Select the checkboxes next to the attributes you want to prune. 
+1. Go to the **Cardinality Management** page of the account.
+2. From the **Metrics** table, select the metric in which you want to create a pruning rule. The **Cardinality Management** page of the metric opens.
+3. Go to the **Attributes** table and select the attributes you want to prune. 
   <img
```

Comment:
justify this image

Reply:
The "Create Pruning Rule" and "Cancel" buttons appear in the table only when a user selects an attribute. Since these are dynamic buttons, users who are not logged into New Relic One might be confused without this screenshot, as these buttons are not visible in the initial Cardinality Management page screenshot.